app/config/__init__.py

```diff
@@ -98,15 +98,14 @@ def SQLALCHEMY_ENGINES(self) -> dict[str, str]:
         "flask_debugtoolbar.panels.g.GDebugPanel",
     ]
 
-    # Flask-Vite
-    VITE_AUTO_INSERT: bool = False
-    VITE_FOLDER_PATH: str = "app/vite"
-
     # GOV.UK Notify
     GOVUK_NOTIFY_DISABLE: bool = False
     GOVUK_NOTIFY_API_KEY: str
     GOVUK_NOTIFY_MAGIC_LINK_TEMPLATE_ID: str = "9773e73c-85a1-4c3f-a808-02b9623616a3"
 
+    ASSETS_VITE_BASE_URL: str = "http://localhost:5173"
+    ASSETS_VITE_LIVE_ENABLED: bool = False
```

Comment:
Could this be local-only config? I think the idea of this ever being able to be turned on in a deployed environment feels odd maybe.

Reply:
But if the principle we're sticking to is "all config must be declared on the base class" - I can also get behind that.